### PR TITLE
Cast correlation_id to str + removed unused imports + updated comments

### PR DESCRIPTION
**Pull request by rantonmattei**
@mortaliorchard @jessmchung @theorchard/python-reviewers 

Regarding the `str` casting, this is because having the correlation as something different from an `str` would cause the payload not to be encoded properly. The handler would silently fail. I ended up with no record in Loggly when I was passing `uuid.uuid1()` as `correlation_id`

### CODE REVIEW

File: owslogger/logger.py [hunk 1]
```diff
@@ -159,7 +162,6 @@ def emit(self, record):
                 'timestamp': utc_date.isoformat(),
                 'level': level_number,
                 'level_name': level_name,
-                'correlation_id': record.correlation_id,
```
Comment (mortaliorchard) [inline owslogger/logger.py]: Correlation id should always be here. It is intended to fail if not provided.

Comment (rantonmattei) [inline owslogger/logger.py]: removed the check and added str cast here

**Comment by rantonmattei** [comment]
Talked to @mortaliorchard about the correlation_id being "required". Would be nice to add proper validation if you want to enforce it. It is not very clear.

**Comment by rantonmattei** [comment]
@jessmchung

**Comment by mortaliorchard** [comment]
R+

**Comment by rantonmattei** [comment]
Can you merge? I don't have access.
@jessmchung @mortaliorchard

**Comment by rantonmattei** [comment]
@mymac80 ^^^

**Pull request merged**